### DISCUSSION TITLE
Welcome to Fonoster Discussions

### DISCUSSION THREAD

**Discussion by psanders** [fonoster/yaps#204] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about
- Share ideas
- Engage with other community members
- Welcome others and be open-minded. Remember that this is a community we build together 💪

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sarikarathod20** [comment]
Hi Pedro, Sarika here I am Full stack developer at mnc company in india and I was doing some R&D on managing calls and stuff and landed on your project, I did all the stuff asked but when I am trying to initiate a call it says permission denied, any insight here ? what could be the reason of getting code 7 permission denied.

**Comment by sossou19** [comment]
bonjour je n'arrive pas a installer fonoster dans mon invite de commande
![fonoster](https://github.com/fonoster/fonoster/assets/132782737/fe46bee5-b486-462b-bcad-121becef5893)
